pom.xml

```diff
@@ -105,6 +105,18 @@
       <artifactId>xml-resolver</artifactId>
       <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <!--scope>test</scope-->
```

Comment:
To continue with my tck testing, I pushed a snapshot build of this xerces change and wildfly-core.  I hacked wildfly-core to ignore the junit dependency but still think this should be addressed.